src/config/configurations.rs

```diff
@@ -0,0 +1,116 @@
+    pub const V1: Configuration = Configuration {
+        transport_strategy: TransportStrategy {
+            grpc_configuration: GrpcConfiguration {
+                deadline_millis: Duration::from_millis(1100),
+                keep_alive_while_idle: true,
+                keep_alive_interval: Duration::from_secs(5000),
+                keep_alive_timeout: Duration::from_secs(1000),
+            },
+        },
+    };
+}
+
+/// This config prioritizes keeping p99.9 latencies as low as possible, potentially sacrificing
+/// some throughput to achieve this. Use this config if low latency is more important in
+/// your application than cache availability.
+pub mod low_latency {
+    use crate::config::configuration::Configuration;
+    use crate::config::grpc_configuration::GrpcConfiguration;
+    use crate::config::transport_strategy::TransportStrategy;
+    use std::time::Duration;
+
+    /// Latest recommended config for a low-latency environment.
+    ///
+    /// NOTE: this config may change in future releases to take advantage of improvements
+    /// we identify for default configurations.
+    pub const LATEST: Configuration = V1;
+
+    /// V1 config for a low-latency environment.
+    ///
+    /// This config is guaranteed not to change in future releases of the Momento Rust SDK.
+    pub const V1: Configuration = Configuration {
+        transport_strategy: TransportStrategy {
+            grpc_configuration: GrpcConfiguration {
+                deadline_millis: Duration::from_millis(500),
+                keep_alive_while_idle: true,
+                keep_alive_interval: Duration::from_secs(5000),
+                keep_alive_timeout: Duration::from_secs(1000),
+            },
+        },
+    };
+}
+
+/// Provides defaults suitable for a typical lambda environment. It has more aggressive timeouts
+/// than the laptop config and does not check connection health with a keep-alive.
+pub mod lambda {
+    use crate::config::configuration::Configuration;
+    use crate::config::grpc_configuration::GrpcConfiguration;
+    use crate::config::transport_strategy::TransportStrategy;
+    use std::time::Duration;
+
+    /// Latest recommended config for a typical lambda environment.
+    ///
+    /// NOTE: this config may change in future releases to take advantage of improvements
+    /// we identify for default configurations.
+    pub const LATEST: Configuration = V1;
+
+    /// V1 config for a typical lambda environment.
+    ///
+    /// This config is guaranteed not to change in future releases of the Momento Rust SDK.
+    pub const V1: Configuration = Configuration {
+        transport_strategy: TransportStrategy {
+            grpc_configuration: GrpcConfiguration {
+                deadline_millis: Duration::from_millis(1100),
+                keep_alive_while_idle: false,
+                keep_alive_interval: Duration::MAX,
+                keep_alive_timeout: Duration::MAX,
```

Comment:
we need to be really careful with these. I feel pretty confident that the equivalent of `null` or `undefined` for these keepalive settings means something very different than `Duration::MAX`. And I think we want the former here, not the latter.

Reply:
I'll do some testing. I based it off of the .NET sdk, where we use max duration, but assuming that it will behave the same across languages may not be the best plan.